website_lazy_load_image/models/ir_ui_view.py

```diff
@@ -22,7 +22,7 @@ def render_template(self, template, values=None, engine='ir.qweb'):
         website_id = self.env.context.get('website_id')
         if website_id and not \
                 self.env['website'].browse(website_id).is_publisher():
-            html = etree.HTML(res)
+            html = lxml.html.fromstring(res.decode())
```

Comment:
Using py2.7 for odoo 11.0 you can get a error here:
 - ![Screen Shot 2019-07-25 at 7 41 29 PM](https://user-images.githubusercontent.com/6644187/61917728-38489b80-af14-11e9-9e71-8f07a503f6bb.png)


Please, consider adds `.decode(UTF-8)`